src/databricks/labs/ucx/mixins/fixtures.py

```diff
@@ -1183,12 +1187,26 @@ def remove(table: dict):
 
 
 @pytest.fixture
-def make_dbfs_data_copy(ws):
+def make_dbfs_data_copy(ws, make_cluster, env_or_skip):
+    if ws.config.is_aws:
+        cluster = make_cluster(
+            single_node=True,
+            instance_pool_id=env_or_skip("TEST_INSTANCE_POOL_ID"),
+            aws_attributes=compute.AwsAttributes(instance_profile_arn=env_or_skip("TEST_WILDCARD_INSTANCE_PROFILE")),
+        ).result()
+        cmd_exec = CommandExecutor(ws.clusters, ws.command_execution, lambda: cluster.cluster_id)
```

Comment:
```suggestion
        cmd_exec = CommandExecutor(ws.clusters, ws.command_execution, lambda: env_or_skip("TEST_DEFAULT_CLUSTER_ID"))
```
Why do we have to wait for nodes to free up if we have a running cluster?...